hatchery/nextflow.go

```diff
@@ -1075,11 +1082,33 @@ func setupSubnet(subnetName string, cidr string, vpcid string, ec2Svc *ec2.EC2)
 		return exsubnet.Subnets[0].SubnetId, nil
 	}
```

Comment:
Here, can we instead of just checking for Subnet > 0, can we also verify whether the existing subnets have an AZ of 1e and return the subnet Id only when it is not 1e?